index.html

```diff
@@ -307,7 +318,42 @@ <h3 class="contact-me">I'm always interested in hearing about new projects, so i
     </footer>
 
 </section>
+</div>
+<div class="modal" id="modal">
+    <div class="modal-header">
+        <div>
+            <h1 class="mod-header">Multi Post Stories</h1>
+            <h1 class="mod-header1">Keeping track of hundreds  of components website</h1>
+        </div>
+    <div>
+            <button data-close-button id="mod-button" class="fa-solid fa-xmark" ></button>
+    </div>
+    </div>
+    <div class="mod-btn-container">
+    <ul class="mod-buttons">
+        <li><button class="mod-btn" type="button">HTML</button></li>
+        <li><button class="mod-btn" type="button">Bootstrap</button></li>
+        <li><button class="mod-btn" type="button">Ruby on Rails</button></li>
+    </ul>
+    </div>
+    <div class="images-popup">
+        <img src="images/Snapshoot-Portfolio.png" alt="IOT-10-image" class="mod-img">
+        <img src="images/Snapshoot-Portfolio1.png" alt="IOT-10-image2" class="mod-img1">
```

Comment:
- [ ] Currently, you have one project with a modal. Please add a modal for all your projects.

<img width="556" alt="image" src="https://user-images.githubusercontent.com/52934128/231571797-7e7b5d47-e356-4fab-b343-55beded1e45b.png">